packages/react-components/src/localization/localization.ts

```diff
@@ -0,0 +1,33 @@
+import { CameraButtonStrings } from '../components/CameraButton';
+
+interface LocalizationProps {
+  /**
+   * Specific strings overides for components.
+   *
+   * Values here override those provided via the locale.
+   */
+  strings?: LocalizationStrings;
```

Comment:
Is this structure able to treeshake unused strings?

Reply:
Discussed IRL: Plan is to keep all strings / nested objects optional.